### ISSUE TITLE
Contributing

### ISSUE THREAD

**Issue by elovelan** [KNOXDEV/wsl#1]
Since this is a mirror of https://git.irs.sh/KNOXDEV/wsl, should PRs be created there? If so, how? Would you be willing to create a [CONTRIBUTING](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) file to answer this (and other) contribution questions?

**Comment by KNOXDEV** [comment]
Sure, done. I just used a template, so the document itself is a bit wordy: https://github.com/KNOXDEV/wsl/blob/master/CONTRUBUTING.md

I'll save you the trouble: yes, PR's are totally welcome. Create them here. I'm phasing out the **git.irs.sh** server.
